README.md

```diff
@@ -11,8 +11,22 @@ Uses SQLite to keep data in one file, so you need this external go dependency
 ```
 go get github.com/mattn/go-sqlite3
 ```
+Other than that, no external dependencies are needed.
 
-Other than that, no external dependency is needed.
+To run the app navigate to the /stuff directory and run:
+
+```
+go run main.go dbbackend.go search-handler.go search.go form-handler.go image-handler.go resistor-image.go template-renderer.go status-handler.go sitemap-handler.go
+```
+
+Or you can navigate to the /stuff directory and run:
+
+```
+make all
```

Comment:
Cool.  I figured out the long go run command just by trial and error trying to get it to run never having made a go thing run before.  But those are good tips and I'll change it just use make stuff.  I should probably note they should be working in a Bash or similar shell for this to work?